tests/EEGPSR_appendix.tex

```diff
@@ -0,0 +1,144 @@
+\chapter[EEGPSR in detail]{E\textsuperscript{2}GPSR in detail.}
+\label{chap:eegpsr-appendix}
+
+The following section presents detailed examples and descriptions of the skills to be evaluated during the \textit{EEGPSR} test (see \refsec{sec:eegpsr}). Keep in mind that this is provided only as a guideline to illustrate what teams can expect from the test, but it sets no constrains on how the test will be addressed.
+
+\section{Ability examples}
+\begin{itemize}
+	\item \textbf{Advanced manipulation:} The ability of manipulating objects with \textit{two hands} or that require precision and coordination.
+	\begin{itemize}
+		\item Open a bottle (twist, uncap, etc.).
+		\item Pouring cereal in a bowl.
+		\item Placing objects inside a box
+		\item Transporting a tray.
+	\end{itemize}
+
+	\item \textbf{Activity detection:} The ability of detecting human activities is evaluated. It may be requested on purpose such as when the robot is commanded to awake someone, or be triggered by an unexpected event like when a person faints.
+	\begin{itemize}
+		\item A dormant person wakes up.
+		\item People is dancing, reading a book.
+	\end{itemize}
+
+	\item \textbf{Awareness (see \refsec{sec:eegpsr-awarenes}):} The ability of detecting \textit{unexpected} events and reacting accordingly.
+	\begin{itemize}
+		\item A door on the robot's path is shut.
+		\item The operator calls the robot by waving.
+		\item The person being followed by the robot faints.
+		\item The person being looked for is sitting on the couch.
+		\item The person being looked for is found lying on the floor.
+		\item Someone stands up in front of the robot blocking its path.
+		\item Someone enters the room and sits down or stands up and walks away while the robot is counting people in a room.
+	\end{itemize}
+
+	\item \textbf{Collision-free navigation:} The ability of navigating safely without colliding (even slightly) with the environment. This ability is not commanded but supervised.
```

Comment:
Changed that to:

    \item \textbf{Collision-free navigation:} The ability of navigating safely
    without colliding (even slightly) with the environment. This ability is not
    commanded, in the sense that the robot is not asked to avoid collisions,
    but evaluated by the referees while the robot is moving.